components/features/Dashboard/Applications/InteractiveQuiz/Assignment/Assignment.tsx

```diff
@@ -10,25 +11,14 @@ interface Props {
   >;
 }
 
-const getSecondsRemaining = (time: string) => {
-  const timeinList = time.split(":").map((x) => parseInt(x));
-  const [hours, minutes] = timeinList;
-  let seconds: number = 0;
-  if (timeinList.length > 2) seconds = timeinList[2];
-  const date = new Date();
-  date.setUTCHours(hours);
-  date.setUTCMinutes(minutes);
-  date.setUTCSeconds(seconds);
-  const diffMs = date.getTime() - new Date().getTime();
-  const diffSeconds = Math.floor(diffMs / 1000)
-  return diffSeconds;
-};
 
 const Assignment: React.FC<Props> = ({
   assignment,
   setCurrentActiveAssignment,
 }) => {
-  const durationInSeconds = getSecondsRemaining(assignment.end_working_time);
+  const durationInSeconds = dateStringToSeconds(assignment.end_working_time);
```

Comment:
Hello there, Rashad. Is the functionality of the functions changing? Or is it just a name change? Qapla!

Reply:
Thanks for the clarification, Rashad.